src/flypad-backend/src/Aircraft/AircraftPreset.cpp

```diff
@@ -38,8 +38,8 @@ void AircraftPreset::onUpdate(double deltaTime) {
     // needs to be initialized
     if (!loadingIsActive) {
       // check if procedure ID exists
-      vector<ProcedureStep*>* requestedProcedure = procedures.getProcedure(loadAircraftPresetRequest);
-      if (requestedProcedure == nullptr) {
+      const std::vector<const ProcedureStep*>* requestedProcedure = procedures.getProcedure(loadAircraftPresetRequest);
+      if (requestedProcedure->empty()) {
```

Comment:
This would create a runtime error if requestedProcedure is nullptr.
getProcedure will return a nullptr when loadAircraftPresetRequest != 1|2|3|4|5

```suggestion
      if (requestedProcedure == nullptr) {
```